website/templates/emails/initial_confirm.txt.mako

```diff
@@ -0,0 +1,10 @@
+Hello ${user.fullname},
+
+Thank you for registering for an account on the Open Science Framework.
+
+Please verify your email address by visiting this link:
+
+${confirmation_url}
+
+The OSF Team
```

Comment:
There will need to be blank line here in order for there to be a newline in the rendered email. Otherwise, the email will look like

```
The OSF Team Center for Open Science
```

I can do this when I merge.